lib/src/signed_video_h26x_sign.c

```diff
@@ -186,24 +186,26 @@ generate_sei_nalu(signed_video_t *self, uint8_t **payload, uint8_t **payload_sig
   const size_t hash_size = sign_data->hash_size;
 
   // Metadata + hash_list forming a document.
-  const sv_tlv_tag_t document_encoders[] = {
+  const sv_tlv_tag_t golden_sei_document_encoders[] = {PUBLIC_KEY_TAG, PRODUCT_INFO_TAG};
+  const sv_tlv_tag_t gop_info_encoders[] = {
```

Comment:
I would prefer to keep this declared last as before, since it by definition should end up last in the SEI.